website/pages/en/sps/sps-intro.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Introduction
+---
+
+By leveraging a Substreams package (`.yaml`) as a data source, your subgraph gains access to pre-extracted, indexed blockchain data, enabling more efficient and scalable data handling, especially when dealing with large or complex blockchain networks.
+
+This technology opens up more efficient and versatile indexing for diverse blockchain environments. For more information on how to build a Substreams-powered subgraph, [click here](./triggers.mdx). You can also visit the following links for How-To Guides on using code-generation tooling to scaffold your first end to end project quickly:
```

Comment:
Two things about the link:
1. We should not include the `.mdx` part ideally... it looks like some code somewhere catches that and excludes it from the rendered link, but we shouldn't rely on that.
2. I am torn about relative links. I think same-folder ones are fine, but I prefer absolute (well, app-relative, e.g. `/sps/triggers`, not fully absolute) in general because it's more greppable.